doc/historylog.rst

```diff
@@ -2,26 +2,48 @@
 History
 #######
 
-With the **History** pane, you can view an automatically de-duplicated, time-stamped list of every command you enter into any connected :doc:`ipythonconsole`.
+With the **History** pane, you can view all the commands entered into any :doc:`ipythonconsole` along with their time-stamp.
 
 .. image:: images/history/history-menu.png
    :alt: Spyder History Log, displaying a list of previously executed commands
 
+(Replace screenshot for screenshot without the options menu)
 
 
-=====================
-Using the History Log
-=====================
 
-Navigating the :guilabel:`History Log` is very straightforward.
-Each session is marked by a date and time-stamp, making it easy to remember when you executed a certain command.
+=================
+Using the History
+=================
```

Comment:
Needs "pane" here to be clear that its referring to the pane, rather than the history shown by the pane, a usage which matches e.g. the Find pane docs.

```suggestion
======================
Using the History pane
======================
```